test/functional/feature_privatesend.py

```diff
@@ -7,7 +7,7 @@
 from test_framework.util import assert_equal
 
 '''
-privatesend.py
+feature_privatesend.py
```

Comment:
This one should be named `rpc_privatesend.py` imo

Reply:
https://github.com/dashpay/dash/pull/3612/commits/0d2e6b104fe70d716ce60a7cd40d83f1f87f3ecd